src/components/GameItem.js

```diff
@@ -2,28 +2,57 @@ import React, { Component } from 'react';
 import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
+// import App from '../src/App.js'
+
 
 class GameItem extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      _checked: false,
+      _divClass: "game-item"
+    }
```

Comment:
Generally you should only prefix variable names with `_` if you're writing getters and setters for them.